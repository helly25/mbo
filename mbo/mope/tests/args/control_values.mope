Start: {{config_start}}
End:   {{config_end}}
Step:  {{config_step}}
Join:  '{{config_join}}'
List:  {{#list=config_start;config_end;config_step;config_join}}{{list}}{{/list}}
Nested:{{#list=config_start;config_end;config_step;'|'}}{{#inner=list;config_end;config_step;','}}{{inner}}{{/inner}}{{/list}}
List:  {{#list=config_start;config_end;config_step;config_join}}{{list}}{{/list}}
Nested:{{#list=config_start;config_end;config_step;'|'}}{{#inner=list;config_end;config_step;','}}{{inner}}{{/inner}}{{/list}}
