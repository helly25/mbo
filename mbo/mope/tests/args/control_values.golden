Start: 12
End:   42
Step:  10
Join:  '+'
List:  12+22+32+42
Nested:12,22,32,42|22,32,42|32,42|42
List:  12+22+32+42
Nested:12,22,32,42|22,32,42|32,42|42
